index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Your description goes here">
+    <meta name="keywords" content="one, two, three">
+    <title>Celsius Converter</title>
+
+    <!-- external CSS link -->
+   
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="title">
+        <h1>Celsius Converter</h1>
+      </div>
+    </header>
+      <div class="container">
+        <form>
+          <input type="number" id="ctemp" placeholder="Enter your temperature in celsius">
```

Comment:
@jcooper8190 when you have an input with the type of `number` you get that controller on the right  side of the field to increment through number values. Even thought this mitigates you having to change the string value to a number. The final UX is probably not what you would want.